functions/src/repository/response-git-webhook-repository.ts

```diff
@@ -104,6 +105,7 @@ export const onResponseGitWebhookRepository: HttpsFunction = https.onRequest((re
         .then(() => {
           Logger.info('Parsing done!');
           res.status(200).send();
+          onCreateEvent();
```

Comment:
Not sure if the `res...send()` has to come last?

Reply:
I think you are right. `res...send()` should come at the last after updating the event count. I will update it